test/controllers/ReturnSentControllerSpec.scala

```diff
@@ -17,30 +17,550 @@
+      val application = applicationBuilder(Some(userAnswers), Some(returnPeriod)).overrides(
+        bind[SDILSessionCache].toInstance(mockSessionCache),
+        bind[SoftDrinksIndustryLevyConnector].toInstance(mockSdilConnector)
+      ).build()
+
+      running(application) {
+        val request = FakeRequest(GET, routes.ReturnsController.onPageLoad(false).url)
+        val result = route(application, request).value
+
+        status(result) mustEqual OK
+        val page = Jsoup.parse(contentAsString(result))
+        page.getElementById("amountDue" ).text must include(Messages("checkYourAnswers.amountToPay.title"))
+        page.getElementsByTag("dt").text() must include(Messages("balanceBroughtForward.checkYourAnswersLabel"))
+        page.getElementsByTag("dd").text() must include("£0.00")
+        page.getElementsByTag("dd").text() must include("£660.00")
+      }
+    }
+
+    "must show correct total when non nil return" in {
+      val amounts = Amounts(660, 0, 660)
+      when(mockSessionCache.fetchEntry[Amounts](any(), any())(any())) thenReturn Future.successful(Some(amounts))
+      val userAnswers = UserAnswers(sdilNumber, Json.obj(), List.empty, Map.empty, true)
+      val application = applicationBuilder(Some(userAnswers), Some(returnPeriod)).overrides(
+        bind[SDILSessionCache].toInstance(mockSessionCache),
+        bind[SoftDrinksIndustryLevyConnector].toInstance(mockSdilConnector)
+      ).build()
+
+      running(application) {
+        val request = FakeRequest(GET, routes.ReturnsController.onPageLoad(false).url)
+        val result = route(application, request).value
+
+        status(result) mustEqual OK
+        val page = Jsoup.parse(contentAsString(result))
+        page.getElementById("amountDue" ).text must include(Messages("checkYourAnswers.amountToPay.title"))
+        page.getElementsByTag("dt").text() must include(Messages("total.checkYourAnswersLabel"))
+        page.getElementsByTag("dd").text() must include("£660.00")
+      }
+    }
+
+    "must display 0 lowband and highband amounts when for own brands and small producer" in {
+      val amounts = Amounts(660, 0, 660)
+      when(mockSessionCache.fetchEntry[Amounts](any(), any())(any())) thenReturn Future.successful(Some(amounts))
+
+      val userAnswers = UserAnswers(sdilNumber, Json.obj(), List.empty, Map.empty, true)
+      val subscription = aSubscription.copy(activity = RetrievedActivity(true, true, false,false,false))
+      val application = applicationBuilder(Some(userAnswers), Some(returnPeriod), Some(subscription)).overrides(
+        bind[SDILSessionCache].toInstance(mockSessionCache),
+        bind[SoftDrinksIndustryLevyConnector].toInstance(mockSdilConnector)
+      ).build()
+
+      running(application) {
+        val request = FakeRequest(GET, routes.ReturnsController.onPageLoad(false).url)
+        val result = route(application, request).value
+
+        status(result) mustEqual OK
+      }
+    }
+
+  }
```

Comment:
Tests are barely covering controller code. I understand the direction has been set to get a returns sent page in for the refactoring to continue so please check with Christine to confirm if this is acceptable for now until we do the refactor.  
![image](https://user-images.githubusercontent.com/5639505/231732146-b555b789-5bcb-499f-ac3f-c4569c39bb49.png)

Reply:
I'll ask  Christine but I think those are being moved in the refactor